doc/nrf/samples/samples_zigbee.rst

```diff
@@ -3,6 +3,78 @@
 Zigbee samples
 ##############
 
+The nRF Connect SDK provides the following samples showcasing the :ref:`ug_zigbee` protocol.
+The samples can be built for various boards and can be configured for different usage scenarios.
+
+In addition to their basic functionalities, the samples can be expanded with variants and extensions.
+The following table lists variants and extensions available for each Zigbee sample:
+
+.. list-table::
+    :widths: auto
+    :header-rows: 1
+
+    * - Variant or extension
+      - Light bulb
+      - Light switch
+      - Network coordinator
+      - NCP
+      - Shell
+      - Template
+    * - FEM support
+      - ✔
+      - ✔
+      - ✔
+      - ✔
+      - ✔
+      -
+    * - Sleepy End Device behavior
```

Comment:
This is a feature of the End Device role, only Light Switch sample implements that role.
If we assume that `**` includes changing the device role for given sample, I would include `light bulb` sample too - it doesn't make sense from end-product perspective but it's fine from configuration perspective.

Reply:
I do not agree that making a light bulb a SED is as simple as it is with the Shell or template.
For Light Bulb - you have to modify the sample code to change the role and add SED config. For Shell and template, setting the role is just a matter of Kconfig options and only the SED config has to be added.
@sebastiandraus : Would you agree with adding `***` for the Light Bulb?

Reply:
OK, extended to light bulb.

Reply:
Agree, `***` for Light Bulb - not that simple of the change but still possible.